pkg/manager/member/tikv_failover_test.go

```diff
@@ -120,6 +125,7 @@ func TestTiKVFailoverFailover(t *testing.T) {
 			expectFn: func(t *testing.T, tc *v1alpha1.TidbCluster) {
 				g := NewGomegaWithT(t)
 				g.Expect(len(tc.Status.TiKV.FailureStores)).To(Equal(1))
+				g.Expect(tc.Status.TiKV.FailoverUID).To(BeEmpty())
```

Comment:
Why it's empty with 1 failure store?

Reply:
The UID be generated together with append FailureStores